scripts/check_samples.py

```diff
@@ -42,40 +42,33 @@ def build_sample_set():
                 sys.exit(1)
     return sample_set
 
-def check_template_sample(template_file_name, sample_set):
+def check_template_sample(template_object, sample_set):
     """
     check_template_sample checks if a template has a sample associated in the samples/ folder
 
     Args:
-        template_file_name: the template file path to check
+        template_object: the template dict to check, which got parsed from the file content
         sample_set: set of sample kinds (string) built by build_sample_set 
     Raises:
         yaml.YAMLError if the input template or any sample file is not a valid YAML file
     Returns:
         Boolean - True if a sample was found for this template, False otherwise.
     """
-
+ 
```

Comment:
Whitespace error.

Reply:
I am not sure about spaces here or not. I updated the whole script to remove all warnings from pylint3.